avm/res/network/azure-firewall/tests/e2e/addpip/main.test.bicep

```diff
@@ -0,0 +1,84 @@
+targetScope = 'subscription'
+
+metadata name = 'Add-PIP'
+metadata description = 'This instance deploys the module and attaches an existing public IP address.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-network.azurefirewalls-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'nafaddpip'
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableTelemetry bool = true
```

Comment:
should not be in this or any other test file

Reply:
done